.eslintrc.js

```diff
@@ -1,18 +1,16 @@
 const baseline = require('@mui/monorepo/.eslintrc');
 const path = require('path');
 
-const chartsPackages = ['x-charts', 'x-charts-pro'];
-
-const dataGridPackages = [
+const CHARTS_PACKAGES = ['x-charts', 'x-charts-pro'];
+const GRID_PACKAGES = [
   'x-data-grid',
   'x-data-grid-pro',
   'x-data-grid-premium',
   'x-data-grid-generator',
 ];
-
-const datePickersPackages = ['x-date-pickers', 'x-date-pickers-pro'];
-
-const treeViewPackages = ['x-tree-view', 'x-tree-view-pro'];
+const PICKERS_PACKAGES = ['x-date-pickers', 'x-date-pickers-pro'];
+const TREE_VIEW_PACKAGES = ['x-tree-view', 'x-tree-view-pro'];
+const SCHEDULER_PACKAGES = ['x-tree-view', 'x-tree-view-pro'];
```

Comment:
Are these names intentional or a copy-paste mistake? 🤔 
```suggestion
const SCHEDULER_PACKAGES = ['x-scheduler', 'x-scheduler-pro'];
```

Reply:
```suggestion
const SCHEDULER_PACKAGES = ['x-scheduler'];
```


I just forgot THE important piece :laughing: